mitodl/mit-open
# update unit names

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4748

### Description (What does it do?)
This PR updates unit names in `offered_by.json` and `platforms.json` according to what is described in the attached issue.

### How can this be tested?
 - Spin up `mit-open` on this branch
 - Run the following commands, one at a time:
```
docker compose exec web ./manage.py loaddata platforms offered_by
docker compose exec web ./manage.py update_offered_by
docker compose exec web ./manage.py update_platforms
docker compose exec web ./manage.py backpopulate_resource_channels --overwrite
docker compose exec web ./manage.py recreate_index --all
```
 - After all of the commands above have been run successfully, test the following:
   - Visit http://localhost:8063/units
   - Click any of the units described in the issue above and verify that the channel page has been updated with the new information
   - Visit http://localhost:8063/search and scroll down to the "Offered By" facet and expand it
   - Verify that the names have been updated in the facet
   - Filter by the facets one at a time, and in each one click a card and verify that the name has been updated in the learning resource drawer
   - Visit http://localhost:8063/api/v1/learning_resources_search/?limit=12&sortby=new and verify that the `platforms` and `offered_by` sections include the updated names

### Checklist:
- After the code is deployed to each environment, run the following in a Heroku shell (`heroku run -a <app name> /bin/bash`):
  - [ ] `./manage.py loaddata platforms offered_by`
  - [ ] `./manage.py update_offered_by`
  - [ ] `./manage.py update_platforms`
  - [ ] `./manage.py backpopulate_resource_channels --overwrite`
  - [ ] `./manage.py recreate_index --all`
